NAMESPACE

```diff
@@ -111,6 +113,7 @@ importFrom(shiny,addResourcePath)
 importFrom(shiny,getShinyOption)
 importFrom(shiny,includeScript)
 importFrom(shiny,tags)
+importFrom(tools,file_ext)
```

Comment:
Can you use the internal file_ext() function(in utils.R) instead ? This allows to remove tools from deps

Reply:
I remove the importFrom from both `NAMESPACE` and `add_files.R`